convert2rhel/pkghandler.py

```diff
@@ -608,6 +609,32 @@ def remove_non_rhel_kernels():
     return non_rhel_kernels
 
 
+def fix_default_kernel():
+    """
+        On systems converted from OL 6 and latter or CentOS Plus, DEFAULTKERNEL may contain kernel-uek/kernel-plus
+        after the conversion.
+
+        The solution handled by this function is to remove the  kernel-uek/kernel-plus from DEFAULTKERNEL setting the
+        correct kernel according with the rhel version.
+        """
+    if system_info.arch == "s390x":
+        # Applicable only on systems using GRUB2 (s390x uses zipl)
+        return
+
+    loggerinst = logging.getLogger(__name__)
+
+    loggerinst.info("Checking remaining Oracle Linux/CentOS Plus kernel and fix to RHEL kernel...")
+    kernel_sys_cfg = utils.get_file_content("/etc/sysconfig/kernel")
+
+    kernel_to_change = ["kernel-uek", "kernel-plus"]
+    # need to change to "kernel" in rhel6, 7 and "kernel-core" in rhel8
+    new_kernel_str = "DEFAULTKERNEL=" + ("kernel" if int(system_info.version) in [6, 7] else "kernel-core")
```

Comment:
Version was changed to dict in since this PR
```suggestion
    new_kernel_str = "DEFAULTKERNEL=" + ("kernel" if system_info.version.major in [6, 7] else "kernel-core")
```